src/services/ContentService.js

```diff
@@ -53,14 +53,16 @@ define(["structures/ContentCreateStruct", "structures/ContentUpdateStruct", "str
      * @method loadRoot
      * @param callback {Function} callback executed after performing the request (see
      * {{#crossLink "ContentService"}}Note on the callbacks usage{{/crossLink}} for more info)
+     * @param xhrCallbacks {Object} a set of callbacks to apply on a specific XHR event like onload, onerror, onprogress, etc.
      */
-    ContentService.prototype.loadRoot = function (callback) {
+    ContentService.prototype.loadRoot = function (callback, xhrCallbacks) {
```

Comment:
In your opinion, which methods should be allowed to handle that param?
There's a lot of them and I don't see a point to guess which one is should have it.
Maybe, we should limit the usage of callbacks to `create*` methods? What do you think?